### PR TITLE
Port to usize/isize in libsyntax

### PR DESCRIPTION
**Pull request by pshc**
Collaboration with @rylev!

I didn't change `int` in the [quasi-quoter](https://github.com/pshc/rust/blob/99ae1a30f3ca28c0f7e431620560d30e44627124/src/libsyntax/ext/quote.rs#L328), because I'm not sure if there will be adverse effects.

Addresses #21095.

### CODE REVIEW

File: src/libsyntax/ast.rs [hunk 1]
```diff
@@ -181,9 +181,9 @@ impl Name {
         }
     }
 
-    pub fn uint(&self) -> uint {
+    pub fn uint(&self) -> usize {
```
Comment (sfackler) [inline src/libsyntax/ast.rs]: This should be renamed to usize.

**Comment by pshc** [comment]
@sfackler Okay, changed and squashed into d5c83652b33a6e5049699ccc7e6bd6fffb42c2b8.

**Comment by sfackler** [comment]
Thanks!

**Comment by pshc** [comment]
D'oh. Sorry about that. Why didn't my `make check` pick that failure up? (I wonder what I missed.)

I've pushed 35dd579cb7c5fb0af48bfc89ac2017864eebb548 which I think should fix it, although I'm not 100% on that. Running tests now.

**Comment by sfackler** [comment]
I don't think you need java and antlr to reach that failure. It looks like it's coming from libsyntax's test, so `make check-stage1-syntax` should run it for you.

**Comment by pshc** [comment]
Yeah, I was pretty confused back there. Okay. I realized that the problem was I had messed with whitespace and therefore the spans of that parse output. So I reverted the whitespace change and fixup'd it into 7a24b3a4d7769ad9a4863a2cc61c009056459a67. Force pushed the rest. Everything should be green now!

**Pull request merged**